exporter/redis.go

```diff
@@ -467,6 +479,24 @@ func (e *Exporter) scrapeRedisHost(scrapes chan<- scrapeResult, addr string, idx
 		}
 	}
 
+	if reply, err := c.Do("LATENCY", "LATEST"); err != nil {
```

Comment:
I don't think the error is that useful. How about inverting the `if` statement and check for `err == nil` and then get rid of the `else` ?